### ISSUE TITLE
is it possible to charge monthly per team

### ISSUE THREAD

**Issue by andrewfreddin** [laravel/spark#197]
I would like to create an application that does not have pre-defined monthly billing amounts. 
To use the example of a bug tracker (the example used in the laracon 2015 keynote, 10m 13s in) , I want to have two teams, a team for my main business, and a team for my side project. 

I want to charge both of these teams separately, with each team itemized in the automatically generated invoice. So if the subscription costs 10 dollars a month, when I add a second team for my side project, the billing information is updated, and I am now being charged 20/month. My invoice is updated accordingly. 

Is this possible currently? Thanks in advance

**Comment by nateritter** [comment]
Nope.  Billing by team is not currently supported, thus the other more specific requests are not either. 

However, the billing by team request is a duplicate of https://github.com/laravel/spark/issues/196 , already proposed as a feature.

**Comment by andrewfreddin** [comment]
Awesome. Thanks for the reply. After posting this question I did some research and Cashier can do what I need (other than the invoicing) so I have started looking at that route for now.
